third-party/chpl-venv/venv-use-sys-python.py

```diff
@@ -4,32 +4,176 @@
 
 import os
 import sys
+import re
+
+# This fixes up the chpl-venv installation in a Cray Chapel module that was built using a non-standard
```

Comment:
Sorry, wrong word/link -- I meant a [module doc string](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings).

Basically wrap this with `"""`s, and add a 1 line summary of the module (python script) at the top of it.